relay-server/src/actors/processor.rs

```diff
@@ -2278,6 +2279,24 @@ impl EnvelopeProcessorService {
             state.managed_envelope.envelope_mut().add_item(item);
         };
 
+        let span_op_db_module_denylist: HashSet<String> = HashSet::from([
+            "db.clickhouse".into(),
+            "db.redis".into(),
+            "db.sql.query".into(),
+        ]);
+        let span_op_browser_module_prefix_allowlist: Vec<String> = vec![
+            "browser".into(),
+            "http".into(),
+            "resource".into(),
+            "ui".into(),
+        ];
```

Comment:
Since these don't change during execution, what do you think about making them `static` and wrapping them in [`OnceCell`](https://doc.rust-lang.org/stable/std/cell/struct.OnceCell.html)? See another [example in Relay](https://github.com/getsentry/relay/blob/b2186c9da45a44d5d39b48e50b8d65b7e592d347/relay-event-normalization/src/normalize/user_agent.rs#L377).